docs/proposals/scheduler/scheduler.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
+
+package scheduler;
+import "google/protobuf/empty.proto";
+
+enum Status {
+    RUNNING = 0;
+    STARTING = 1;
+    STOPPED = 2;
+    STOPPING = 3;
+    DESTROYING = 4;
+    TERMINATED = 5;
+    CRASHED = 6;
+    FAILED = 7;
+    SCHEDULING = 8;
+    SCHEDULED = 9;
+}
+
+enum Action {
+    START = 0;
```

Comment:
Will I have to call `start`after I `create`and instance?

Reply:
Exactly.